### PR TITLE
Added sample scripts for request module blog

### PR DESCRIPTION
**Pull request by kushal-likhi**
Added sample scripts for `request module` blog.

### CODE REVIEW

**Comment by zwigby** [comment]
I might have missed this on the last set but let's make sure we use 2 spaces for indentation.

**Comment by zwigby** [comment]
Let's remove the initial PUT, POST, DELETE, PATCH examples because they are only going to return errors.

File: request/customConfigurations.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    qs: {from: 'blog example', time: +new Date()}, //Query string data
+    method: 'GET',
+    headers: {
+        'Content-Type': 'MyContentType',
+        'Custom-Header': 'Custom Value'
+    }
+}, function(error, response, body){
+    if(error) {
+        console.log(error);
+    } else console.log(response.statusCode, body);
```
Comment (zwigby) [inline request/customConfigurations.js]: Also use { } with if or else. Makes it too easy to break when not using them. This goes for all if/else statements below.

**Comment by kushal-likhi** [comment]
Hi, updated the indentation for all files, also added `.editorconfig` for ensuring 2 spaces via IDE reformats. Also updated the if-else blocks and ensured statements are in blocks, Also updated the blog document with the same.
Removed the PUT/POST/DEL/PATCH code.

**Comment by zwigby** [comment]
Looks like there are conflicts, can you make sure those are resolved. my guess is pull from branch and then submit pull.

**Comment by kushal-likhi** [comment]
Hey! Sorry for that, resolved conflicts and pushed!

**Pull request merged**